app/queries/my_facilities/registrations_query.rb

```diff
@@ -44,6 +44,6 @@ def period_list(period, last_n)
   end
 
   def periods_as_sql_list
-    @periods.map { |(year, day)| "('#{year}', '#{day}')" }.join(',')
+    @periods.map { |(year, period)| "('#{year}', '#{period}')" }.join(',')
```

Comment:
I think you mentioned in an earlier comment that we don't need to destructure the block args with `()`.